tests/utils.py

```diff
@@ -21,6 +21,7 @@ def setup_task(
         overwrite_from_index_token=None,
         state_seq=("in_progress", "finished"),
         op_type="add",
+        deprecation_list=None,
```

Comment:
Should we default this to an empty list instead?

Reply:
No, it is not a good practice adding an empty list to the function parameters.
Anti-pattern: https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html

Reply:
I set the variable in the lines below to the []